lib/clientsidescripts.js

```diff
@@ -47,24 +47,46 @@ function wrapWithHelpers(fun) {
  *     be passed as a parameter.
  */
 functions.waitForAngular = function(rootSelector, callback) {
-  var el = document.querySelector(rootSelector);
-
-  try {
-    if (!window.angular) {
-      throw new Error('angular could not be found on the window');
+  if (!rootSelector) {
```

Comment:
By accidentally I mean if they specify that in their config. The config name itself doesn't mention anything about ng2, and a lot of people don't read the comments.